spec.bs

```diff
@@ -811,6 +819,48 @@ To <dfn>obtain a report's shared info</dfn> given an [=aggregatable report=]
 1. Return the result of [=serializing an infra value to a json string=] given
     |sharedInfo|.
 
+User-agent automation {#user-agent-automation}
+==============================================
+
+For the purposes of user-agent automation and website testing, this document
+defines the below [[WebDriver]] [=extension commands=] to control the API
+configuration.
+
+Set developer mode {#set-developer-mode}
```

Comment:
Shall we align with ARA and use "local testing mode"?

Reply:
Yep, done